### PR TITLE
Some VSCode Errors and Warnings

### PR DESCRIPTION
**Pull request by adamgordonbell**

### CODE REVIEW

File: cmd/earthly/main.go [hunk 1]
```diff
@@ -182,8 +181,6 @@ var (
 var (
 	errLoginFlagsHaveNoEffect            = errors.New("account login flags have no effect when --auth-token (or the EARTHLY_TOKEN environment variable) is set")
 	errLogoutHasNoEffectWhenAuthTokenSet = errors.New("account logout has no effect when --auth-token (or the EARTHLY_TOKEN environment variable) is set")
-	errURLParseFailure                   = errors.New("Invalid URL")
```
Comment (adamgordonbell) [inline cmd/earthly/main.go:185]: unused?

File: cmd/earthly/main.go [hunk 2]
```diff
@@ -3041,25 +3038,6 @@ func (app *earthlyApp) actionBuildImp(c *cli.Context, flagArgs, nonFlagArgs []st
 	return nil
 }
 
-func (app *earthlyApp) hasSSHKeys() bool {
```
Comment (adamgordonbell) [inline cmd/earthly/main.go:3044]: unused?

File: cmd/earthly/main.go [hunk 3]
```diff
@@ -3148,10 +3126,8 @@ func (app *earthlyApp) actionListTargets(c *cli.Context) error {
 			fmt.Printf("+%s\n", t)
 		}
 		if app.lsShowArgs {
-			if args != nil {
```
Comment (adamgordonbell) [inline cmd/earthly/main.go:3151]: https://staticcheck.io/docs/checks#S1031

**Review by alexcb — APPROVED** [review]
nice cleanup!

**Pull request merged**